content/en/docs/faq.md

```diff
@@ -75,3 +75,9 @@ Not natively supported but a user can configure `revocationValidations` to `skip
 **A:** Ideally, we should validate the signing identity first and then use the public key in the signing identity to validate the artifact signature.
 However, this will lead to poor performance in the case where the signature is not valid as there are lots of validations against the signing identity including network calls for revocations, and possibly we won't even need to read the trust store/trust policy if the signature validation fails.
 Also, by validating artifact signature first we will still fail the validation if the signing identity is not trusted.
+
+## Notary Project Terms
+
+**Q: What is the difference between Notary, Notary v2 Notation, TUF, and Notary Project?**
+
```

Comment:
Should we point to the glossary page directly?

Reply:
Yes. `Notary Project Terms` can point to the glossary page directly.